docs/Basics-Layout.md

```diff
@@ -63,7 +63,7 @@ class AwesomeProject {
   }
 };
 
-AppRegistry.registerComponent('AwesomeProject', () => AwesomeProject);
+AppRegistry.registerComponent('AwesomeProject', () => JustifyContentBasics);
```

Comment:
It is built in this way in the web player right now. Maybe down the road it can be configurable...

https://github.com/facebook/react-native/blob/master/website/core/WebPlayer.js#L47